operations/http.go

```diff
@@ -605,6 +606,31 @@ func (ac *legacyClient) PutPatch(incomingPatch patchSubmission) (*patch.Patch, e
 	return reply.Patch, nil
 }
 
+// GetRawPatchWithModules fetches the raw patch and module diffs for a given patch ID
+func (ac *legacyClient) GetRawPatchWithModules(patchId string) (*data.RawPatch, error) {
```

Comment:
I didn't notice this on the previous code review, but it's better to not add any more new routes to the legacy REST client because it's deprecated. Methods should instead be added to the new REST communicator, so we should move this method over.

Reply:
done